opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -265,15 +262,21 @@ def _import_exporters(
     metric_exporters = {}
     log_exporters = {}
 
-    for (exporter_name, exporter_impl,) in _import_config_components(
```

Comment:
here I'll remove the trailing comma in the tuple to keep everything in one line

Reply:
Thanks that worked